easypy/ziplog.py

```diff
@@ -93,11 +144,33 @@ def iter_zipped_logs(*log_streams):
             if ts and ts > current_ts:
                 streams.put((ts, line, stream))
                 break   # timestamp advanced
-            yield line
+            yield formatted(line, ts, last_ts)
+            if ts:
+                last_ts = ts
 
 
-if __name__ == "__main__":
+def main():
     import sys
-    files = map(open, sys.argv[1:])
-    for line in iter_zipped_logs(*files):
-        print(line, end="")
+    import argparse
+    parser = argparse.ArgumentParser(description='ZipLog - merge logs by timestamps')
+    parser.add_argument(
+        'logs', metavar='N', type=str, nargs='+',
+        help='Log files; Use "-" for STDIN')
+    parser.add_argument(
+        '-i', '--interval', dest='interval', default=None,
+        help="Show interval by seconds (s), or milliseconds (ms)")
+    parser.add_argument(
+        '-p', '--prefix', dest='prefix', default="> ",
```

Comment:
I don't like that the CLI tool has different defaults than the command (`> ` vs `DARK_GRAY@{> }@`)

Reply:
but then are you assuming that the cli would be piping to `ezcolorize`?...
I could add a `--colorize` flag, but that would still mean override the default.

regardless, the default should not use coloring, but then you guys may want to update the ziplog calls in teka.

Reply:
I'd rather change a few calls in `teka` than get stuck with weird arguments...

Also, the worst that could happen from neglecting to change `teka` is not having the marker colored.